sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidPlanner.java

```diff
@@ -380,13 +380,7 @@ public static DruidException translateException(Exception e)
         }
       }
 
-      return DruidException.forPersona(DruidException.Persona.DEVELOPER)
```

Comment:
This makes sense to me since this is still a case of `SqlParseException`.

@zachjsh, do you mind adding a test with a bad user-specified SQL that would cause the exception to be targeted towards a user?

Reply:
Added.